### PR TITLE
Add unit tests

### PR DESCRIPTION
**Pull request by semseysandor**
working towards #4 

Tests added for:
- `CRM_Mailchimptemplate_Settings`
- `CRM_Mailchimptemplate_Form_MailchimpTemplateSettings`

### CODE REVIEW

File: tests/phpunit/CRM/Mailchimptemplate/SettingsTest.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        return Test::headless()
+            ->install('rc-base')
+            ->installMe(__DIR__)
+            ->apply();
+    }
+
+    /**
+     * The setup() method is executed before the test is executed (optional).
+     */
+    public function setUp(): void
```
Comment (semseysandor) [inline tests/phpunit/CRM/Mailchimptemplate/SettingsTest.php]: unneeded function

File: tests/phpunit/CRM/Mailchimptemplate/SettingsTest.php [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    {
+        parent::setUp();
+    }
+
+    /**
+     * The tearDown() method is executed after the test was executed (optional).
+     *
+     * This can be used for cleanup.
+     */
+    public function tearDown(): void
```
Comment (semseysandor) [inline tests/phpunit/CRM/Mailchimptemplate/SettingsTest.php]: unneeded function

File: tests/phpunit/CRM/Mailchimptemplate/SettingsTest.php [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+class CRM_Mailchimptemplate_SettingsTest extends TestCase implements HeadlessInterface
+{
+    public function setUpHeadless()
+    {
+        return Test::headless()
+            ->install('rc-base')
+            ->installMe(__DIR__)
+            ->apply();
+    }
+
```
Comment (semseysandor) [inline tests/phpunit/CRM/Mailchimptemplate/SettingsTest.php:21]: please put `setUpBeforeClass()` with `apply(true)` here too

File: tests/phpunit/CRM/Mailchimptemplate/SettingsTest.php (hunk 1 above)
Comment (law909) [reply tests/phpunit/CRM/Mailchimptemplate/SettingsTest.php]: done

File: tests/phpunit/CRM/Mailchimptemplate/SettingsTest.php (hunk 2 above)
Comment (law909) [reply tests/phpunit/CRM/Mailchimptemplate/SettingsTest.php]: done

File: tests/phpunit/CRM/Mailchimptemplate/SettingsTest.php (hunk 3 above)
Comment (law909) [reply tests/phpunit/CRM/Mailchimptemplate/SettingsTest.php:21]: done

**Pull request merged**